hypothesis-python/tests/nocover/test_reusable_values.py

```diff
@@ -52,35 +70,84 @@ def reusable():
     )
 
 
+def is_valid(s):
+    try:
+        s.validate()
+        return True
+    except InvalidArgument:
+        return False
+
+
+reusable = reusable.filter(is_valid)
+
 assert not reusable.is_empty
 
 
-@example(st.integers(min_value=1))
```

Comment:
This explicit example is a duplicate of one of the strategies already in `base_reusable_strategies`, so I removed it.